### DISCUSSION TITLE
Why python?

### DISCUSSION THREAD

**Discussion by Nisthar** [mxrch/GHunt#404] [Q&A] [answered]
This might be a religious question. But why did you choose python and nodejs or others?

**Comment by mxrch** [comment, marked as answer]
Hi !
Because Python is clean, readable, people can contribute easily.
GHunt is more a big "script" than an application / program that needs speed, cross-plateform binaries etc, so I think Python is a good choice for it.

**Reply by Nisthar** [reply]
Great

**Comment by JanitaRiyaz** [comment]
Well as far as this question is concerned... why pyhon ??... because it is most dynamic language.. readable and clean... It has lot of libraries that make our work easy.. and to be honest python is a funny language.. you don't need to be formal.. just go straight and you'll dive into it without any barrier..
